core/common/package.json

```diff
@@ -15,6 +15,7 @@
     "clean": "rimraf lib .rush/temp/package-deps*.json",
     "docs": "betools docs --includes=../../generated-docs/extract --json=../../generated-docs/core/core-common/file.json --tsIndexFile=./core-common.ts --onlyJson",
     "extract-api": "betools extract-api --entry=core-common",
+    "extract-extension-api": "eslint -f visualstudio --rule '\"@itwin/public-extension-exports\":[\"error\",{\"releaseTags\":[\"public\",\"beta\"],\"outputApiFile\":true}]' \"./src/**/*.ts\" 1>&2",
```

Comment:
oh, then you'll need to add the parser options too, so maybe it's better to just have a separate eslintrc file

Reply:
I created a separate eslint config file: "eslint-plugin/dist/configs/extension-exports-config.js". "npm run extact-extension-api" is a lot faster now since its only checking for the one rule. But I'm not too happy with the relative path - is there a better way?